### DISCUSSION TITLE
New Setting

### DISCUSSION THREAD

**Discussion by xxgalaxyexplosionxx** [ScratchAddons/ScratchAddons#4773] [Ideas]
# What should you add to the next scratch addons version?
you should add a "Find variable by name" setting, it would make editing and making projects easier.
# Why?
It makes it easier so with complex projects you can find stuff easier.

**Comment by GarboMuffin** [comment]
Developer tools already has this?

![image](https://user-images.githubusercontent.com/33787854/178041880-9cd2f2eb-f6ce-4e52-b4f5-e40b6bfe0370.png)

**Reply by Endermaster3690** [reply]
i think they suggest finding the actual variables
not the times it was used

**Reply by ghost** [reply]
It does that too...

**Reply by Endermaster3690** [reply]
they mean the one in green, not red
see image
![Screenshot 2022-07-12 172330](https://user-images.githubusercontent.com/80355301/178778459-9ac891c2-d423-4f2b-bc52-fd0c6446bc93.png)
why is there a green dot up there

**Reply by mxmou** [reply]
You can use the popup for adding blocks (middle click).

**Comment by xxgalaxyexplosionxx** [comment]
> You can use the popup for adding blocks (middle click).

Oh ok  I already solved the problem

**Reply by Endermaster3690** [reply]
EXCUSE ME WHAT ARE YOU PASTING

**Reply by GarboMuffin** [reply]
The default Mail app in Windows is horrible and will do that when they reply by email

If you look closely you can see the real message "Oh ok  I already solved the problem"

**Comment by xxgalaxyexplosionxx** [comment]
> EXCUSE ME WHAT ARE YOU PASTING

I already solved it

**Comment by xxgalaxyexplosionxx** [comment]
> It does that too...

Ty but I already solved it, well thank you!

**Comment by xxgalaxyexplosionxx** [comment]
> Developer tools already has this?

I already solved it

**Comment by xxgalaxyexplosionxx** [comment]
> i think they suggest finding the actual variables
> not the times it was used

Sorry I actually solved it but thanks for your help

**Comment by xxgalaxyexplosionxx** [comment]
> The default Mail app in Windows is horrible and will do that when they reply by email
> If you look closely you can see the real message "Oh ok I already solved the problem"

ok

**Comment by xxgalaxyexplosionxx** [comment]
> The default Mail app in Windows is horrible and will do that when they reply by email
> If you look closely you can see the real message "Oh ok I already solved the problem"

Oh ok well I figured out how to do the thing that’s in the… nvm

**Comment by xxgalaxyexplosionxx** [comment]
> EXCUSE ME WHAT ARE YOU PASTING

Wait what

**Comment by xxgalaxyexplosionxx** [comment]
Ok

**Comment by xxgalaxyexplosionxx** [comment]
ok

**Reply by BroJac5246** [reply]
I'm glad your issue was solved!

Unless you have more details, we don't need to continue this discussion anymore :)

Also, I would recommend not replying to GitHub email notifications, and instead responding directly on GitHub. Your email application appears to be doing weird things to your replies, making them harder to read. It's not your fault, but it's a little annoying.
